OracleDatabase/SingleInstance/dockerfiles/18.4.0/checkDBStatus.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: May, 2017
+# Author: gerald.venzl@oracle.com
+# Description: Checks the status of Oracle Database.
+# Return codes: 0 = Database is open and ready to use
+#               1 = Database is not open
+#               2 = Sql Plus execution failed
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# 
+
+POSITIVE_RETURN="OPEN"
+ORACLE_SID="`grep $ORACLE_HOME /etc/oratab | cut -d: -f1`"
+
+# Check Oracle DB status and store it in status
+status=`su -p oracle -c "sqlplus -s / as sysdba" << EOF
+   set heading off;
+   set pagesize 0;
+   select status from v\\$instance;
```

Comment:
As @whisdom said, this should be changed to reflect the `checkStatus.sh` from other CDB databases, i.e. check when the PDB is actually open for read/write.

Reply:
Sound OK for me.